packages/admin-panel/src/table/columnTypes/columnFilters.jsx

```diff
@@ -7,15 +7,41 @@ import React from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import MuiChip from '@material-ui/core/Chip';
-import { Autocomplete, Select } from '@tupaia/ui-components';
+import { Autocomplete, Select, TextField } from '@tupaia/ui-components';
+import { Search } from '@material-ui/icons';
+
+export const DefaultFilter = styled(TextField).attrs(props => ({
+  InputProps: {
+    ...props.InputProps,
+    startAdornment: <Search />,
+  },
+  placeholder: 'Search...',
+}))`
+  margin-block-end: 0;
+  font-size: inherit;
+  width: 100%;
+  min-width: 6rem;
+  max-width: 100%;
+  // The following is overriding the padding in ui-components to make sure all filters match styling
+  .MuiInputBase-input,
+  .MuiInputBase-input.MuiAutocomplete-input.MuiInputBase-inputAdornedEnd {
+    padding-block: 0.6rem;
+    padding-inline: 0.2rem;
+  }
+  .MuiInputBase-root,
+  .MuiAutocomplete-inputRoot.MuiInputBase-adornedEnd.MuiOutlinedInput-adornedEnd {
+    padding-inline-start: 0.3rem;
+  }
+`;
+
+const StyledSelect = styled(Select)``;
```

Comment:
Is this a change that was preparing for a change that didn’t end up materialising?

Looks like `<Select>` can be used directly